src/app/App.js

```diff
@@ -92,11 +79,11 @@ export default class App {
       ? await this._createProxy(contractName)
       : await this._createProxyAndCall(contractClass, contractName, initMethodName, initArgs)
 
-    log.info(` TX receipt received: ${receipt.transactionHash}`)
```

Comment:
For the record the convention afaict was to `log.info("Doing x")` and then `log.info(" Did x")` with the space in the beginning to hint at its relation to the previous log message. (I opened #172 to replace all this by something nicer.)